examples/website-demos/social-messaging/src/components/MessageInput/SocialMessageInput.tsx

```diff
@@ -87,22 +87,22 @@ export const SocialMessageInput = (props: Props) => {
     [text, giphyState, numberOfUploads], // eslint-disable-line
   );
 
-  const handleCommandsClick = () => {
+  const handleCommandsClick = (event: React.MouseEvent) => {
+    event.stopPropagation();
     openCommandsList();
     setGiphyState(false);
     setCommandsOpen(true);
   };
-
   return (
     <>
       <div className='input-ui-container'>
```

Comment:
let's get rid of container entirely as ideally we build upon the class name from parent to child, and in this case container isn't used/appended again